app/src/main/java/org/wikipedia/analytics/eventplatform/Event.kt

```diff
@@ -0,0 +1,27 @@
+package org.wikipedia.analytics.eventplatform
+
+import com.google.gson.annotations.SerializedName
+import org.wikipedia.util.DateUtil.iso8601DateFormat
+import java.util.*
+
+/** Base class for an Event Platform event.  */
+open class Event(private val schema: String, stream: String) {
```

Comment:
It looks like the `schema` parameter needs a `@SerializedName` that is missing here.